src/client/app/+dashboard/dashboard.component.ts

```diff
@@ -1,14 +1,19 @@
 import {Component} from '@angular/core';
-
-/**
- * This class represents the lazy loaded DashboardComponent.
- */
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
 @Component({
   moduleId: module.id,
   selector: 'ap-dashboard',
   templateUrl: 'dashboard.component.html',
   styleUrls: ['dashboard.component.css']
 })
-
-export class DashboardComponent {  
+export class DashboardComponent {
+  loginForm: FormGroup;
+  constructor(private _formBuilder: FormBuilder, private _router: Router) {
+    this.loginForm = this._formBuilder.group({
```

Comment:
how come you have included validations in your page , before accepting code from Sailaja 

@Sailaja-triculin  whats this ?